examples/BUILD

```diff
@@ -107,6 +119,19 @@ cc_binary(
     ],
 )
 
+cc_binary(
+    name = "mtls_greeter_server",
+    srcs = ["cpp/helloworld/mtls_greeter_server.cc"],
+    defines = ["BAZEL_BUILD"],
+    deps = [
+        ":helloworld_cc_grpc",
+        "//:grpc++",
+        "//:grpc++_reflection",
```

Comment:
Is this dependency needed? I think this might get caught by ClangTidy.

Reply:
Yes it's needed, since the server uses `grpc::reflection::InitProtoReflectionServerBuilderPlugin();`.